clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesApiConverter.groovy

```diff
@@ -261,6 +261,39 @@ class KubernetesApiConverter {
       }
     }
 
+    if (container.securityContext) {
+      def securityContext = container.securityContext
+
+      containerBuilder = containerBuilder.withNewSecurityContext()
```

Comment:
Any reason not to use the chaining style from this line down through `withReadOnlyRootFilesystem()`.